CMakeLists.txt

```diff
@@ -4,17 +4,40 @@ project (SyclSTL)
 enable_testing()
 
 option(PARALLEL_STL_BENCHMARKS "Build the internal benchmarks" OFF)
+option(USE_COMPUTECPP "Use ComputeCPP" ON)
 
-message(STATUS " Path to SYCL implementation: ${COMPUTECPP_PACKAGE_ROOT_DIR} ")
 message(STATUS " Path to CMAKE source directory: ${CMAKE_SOURCE_DIR} ")
+set(CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake/Modules/)
 
+if (USE_COMPUTECPP)
+	message(STATUS " Using ComputeCpp CMake")
 
-set(CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake/Modules/)
+	message(STATUS " Path to ComputeCpp implementation: ${COMPUTECPP_PACKAGE_ROOT_DIR} ")
+
+	include(FindOpenCL)
+	include(FindComputeCpp)
+
+	include_directories("${COMPUTECPP_INCLUDE_DIRECTORY}")
+
+else()
+	message(STATUS " Using triSYCL CMake")
 
-include(FindOpenCL)
-include(FindComputeCpp)
+	add_compile_options("-std=c++14") 
```

Comment:
this will break the windows build. 
The correct way of setting the compiler version is using the CXX standard property of the targets:

https://cmake.org/cmake/help/v3.1/prop_tgt/CXX_STANDARD.html